src/data/matrix/basic.lean

```diff
@@ -891,7 +953,7 @@ instance : star_ring (matrix n n α) :=
   star_add := λ M N, by { ext, simp, },
   star_mul := λ M N, by { ext, simp [mul_apply], }, }
 
-@[simp] lemma star_apply (M : matrix n n α) (i j) : star M i j = star (M j i) := rfl
+@[simp] lemma star_apply_of_star_ring (M : matrix n n α) (i j) : star M i j = star (M j i) := rfl
```

Comment:
I'm pretty sure the linter will tell you to remove this lemma, it's strictly weaker than `star_apply`.

Reply:
I removed the old one.